### PR TITLE
Add a method to check for user activity more efficiently

### PR DESCRIPTION
**Pull request by seahawk1986**
than ConfirmShutdown, so the shutdown wrapper won't be called unnecessarily if the user is inactive.

### CODE REVIEW

File: shutdown.c [hunk 1]
```diff
@@ -26,6 +26,12 @@ class cDBusShutdownHelper
     g_variant_builder_unref(builder);
   };
 
+  static void UserActive(cDBusObject *Object, GVariant *Parameters, GDBusMethodInvocation *Invocation)
```
Comment (flensrocker) [inline shutdown.c]: I think the function should be named `IsUserActive`
```suggestion
  static void IsUserActive(cDBusObject *Object, GVariant *Parameters, GDBusMethodInvocation *Invocation)
```

File: README [hunk 2]
```diff
@@ -331,6 +331,11 @@ Interface "shutdown"
   The boolean argument "ignoreuser" will ignore user activity. This may be
   useful for external frontends like XBMC etc..
 
+- ask if user is active
+  vdr-dbus-send.sh /Shutdown shutdown.UserActive
```
Comment (flensrocker) [inline README]:
```suggestion
  vdr-dbus-send.sh /Shutdown shutdown.IsUserActive
```

File: shutdown.c [hunk 3]
```diff
@@ -233,6 +239,9 @@ const char *cDBusShutdownHelper::_xmlNodeInfo =
     "       \"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd\">\n"
     "<node>\n"
     "  <interface name=\""DBUS_VDR_SHUTDOWN_INTERFACE"\">\n"
+    "    <method name=\"UserActive\">\n"
```
Comment (flensrocker) [inline shutdown.c]:
```suggestion
    "    <method name=\"IsUserActive\">\n"
```

File: shutdown.c [hunk 4]
```diff
@@ -272,6 +281,7 @@ void cDBusShutdown::SetShutdownHooksWrapper(const char *Wrapper)
 cDBusShutdown::cDBusShutdown(void)
 :cDBusObject("/Shutdown", cDBusShutdownHelper::_xmlNodeInfo)
 {
+  AddMethod("UserActive", cDBusShutdownHelper::UserActive);
```
Comment (flensrocker) [inline shutdown.c]:
```suggestion
  AddMethod("UserActive", cDBusShutdownHelper::IsUserActive);
```

**Comment by seahawk1986** [comment]
I don't mind changing the method name to IsUserActive - I used the ManualStart method (which also returns a boolean) as a template and tried to follow it's naming scheme.

**Pull request merged**